charts/gardener/controlplane/values.yaml

```
@@ -13,7 +13,7 @@ global:
     logFormat: json
     logVerbosity: "2"
     image:
-      repository: eu.gcr.io/gardener-project/gardener/apiserver
+      repository: europe-docker.pkg.dev/gardener-project/public/gardener/apiserver
```

```suggestion
      repository: europe-docker.pkg.dev/gardener-project/releases/gardener/apiserver
```
The helm charts are for all members of the Gardener community. We should direct them to the image location which includes release artifacts only.